ballerina-tests/tests/01_advanced_type_client.bal

```diff
@@ -15,117 +15,71 @@
+
 type StockQuoteTypedesc typedesc<StockQuote>;
+
 type StockQuotesTypedesc typedesc<StockQuotes>;
+
 type StockNamesTypedesc typedesc<StockNames>;
 
-@test:Config {enable:true}
+@test:Config {enable: true}
 function testHttpsClientInitWithoutSecureSocketConfig() {
-    HelloWorldClient|grpc:Error errorClient = new("https://localhost:9091");
-    if errorClient is grpc:Error {
-        test:assertEquals(errorClient.message(), "The secureSocket configuration should be provided to establish an HTTPS connection");
-    } else {
-        test:assertFail("Secure client initialization without secure socket should fail.");
-    }
+    HelloWorldClient|grpc:Error errorClient = new ("https://localhost:9091");
+    test:assertTrue(errorClient is grpc:Error);
+    test:assertEquals((<grpc:Error>errorClient).message(), "The secureSocket configuration should be provided to establish an HTTPS connection");
 }
 
-@test:Config {enable:true}
-function testSendNestedStruct() {
-    Person p = {name:"Sam", address:{postalCode:10300, state:"Western", country:"Sri Lanka"}};
-    io:println("testInputNestedStruct: input:");
-    io:println(p);
-    string|error unionResp = helloWorldClient->testInputNestedStruct(p);
-    io:println(unionResp);
-    if unionResp is error {
-        test:assertFail(msg = string `Error from Connector: ${unionResp.message()}`);
-    } else {
-        io:println(unionResp);
-        test:assertEquals(unionResp, "Submitted name: Sam");
-    }
+@test:Config {enable: true}
+function testSendNestedStruct() returns grpc:Error? {
+    Person p = {name: "Sam", address: {postalCode: 10300, state: "Western", country: "Sri Lanka"}};
+    string response = check helloWorldClient->testInputNestedStruct(p);
+    test:assertEquals(response, "Submitted name: Sam");
 }
 
-@test:Config {enable:true}
-function testReceiveNestedStruct() {
-    string name  = "WSO2";
-    io:println("testOutputNestedStruct: input: " + name);
-    Person|grpc:Error unionResp = helloWorldClient->testOutputNestedStruct(name);
-    io:println(unionResp);
-    if unionResp is grpc:Error {
-        test:assertFail(msg = string `Error from Connector: ${unionResp.message()}`);
-    } else {
-        io:println(unionResp.toString());
-        test:assertEquals(unionResp.name, "Sam");
-        test:assertEquals(unionResp.address.postalCode, 10300);
-        test:assertEquals(unionResp.address.state, "CA");
-        test:assertEquals(unionResp.address.country, "USA");
-    }
+@test:Config {enable: true}
+function testReceiveNestedStruct() returns grpc:Error? {
+    string name = "WSO2";
+    Person response = check helloWorldClient->testOutputNestedStruct(name);
+    test:assertEquals(response.name, "Sam");
+    test:assertEquals(response.address.postalCode, 10300);
+    test:assertEquals(response.address.state, "CA");
+    test:assertEquals(response.address.country, "USA");
 }
 
-@test:Config {enable:true}
-function testSendStructReceiveStruct() {
+@test:Config {enable: true}
```

Comment:
Do we need to add `enable: true`, since this is enabled by default ?

Reply:
It is just there to enable and disable a certain number of tests at the development time and we just keep it as a practice.